cmake/Modules/FindMKL.cmake

```diff
@@ -367,7 +367,8 @@ ENDIF (MKL_LIBRARIES AND MKL_INCLUDE_DIR)
 
 # Standard termination
 IF(NOT MKL_FOUND AND MKL_FIND_REQUIRED)
-  MESSAGE(FATAL_ERROR "MKL library not found. Please specify library location")
+  MESSAGE(FATAL_ERROR "MKL library not found. Please specify library location \
+    by setting CMAKE_PREFIX_PATH to the root directory of the MKL installation.")
```

Comment:
It might be better to mention that you should append the directory to CMAKE_PREFIX_PATH, in case someone already has directories listed there.